components/shared/loanMaterialModal/LoanMaterialModal.tsx

```diff
@@ -122,30 +144,28 @@ const LoanMaterialModal = ({
         <div className="bg-background-skeleton mx-auto mt-10 mb-5 h-[26px] w-[500px] animate-pulse rounded-full" />
       )}
       {!isLoadingLibraryProfile && !isLoadingLoans && (
-        <p className="text-typo-body-lg mt-10 mb-5 w-full text-center">
-          {!isLoanPossible && (
-            <>
-              Du kan desværre ikke låne flere titler af typen{" "}
-              <span className="font-bold">
-                {`"${getManifestationMaterialTypeSpecific(manifestation)}"`}
-              </span>{" "}
-              i denne måned.
-            </>
+        <div className="mx-auto mt-10 mb-5 w-full max-w-prose space-y-4">
+          <p className="text-typo-subtitle-md text-center">
+            {!isLoadingLibraryProfile &&
+              !isLoadingLoans &&
+              isLoanPossible &&
```

Comment:
could we move `!isLoadingLibraryProfile`and `! isLoadingLoans` into the logic of `isLoanPossible` so we just end up with `isLoanPossible`?

Reply:
You are checking for `errors`later on in the code. Shouldn´t it be part of `isLoanPossible` as well?